indigo/distribution.yaml

```diff
@@ -16323,21 +16323,6 @@ repositories:
       url: https://bitbucket.org/traclabs/trac_ik.git
       version: master
     status: developed
-  tracetools:
```

Comment:
Indigo is EOL we don't accept changes to the released content in the distro anymore. It's standing as a snapshot of the state. 

You can update the URL for a moved repository in case someone is trying to build from source.

Reply:
I tried changing the URL first (see https://github.com/ros/rosdistro/pull/22980/commits/a4e78b6bfb702df3d50cd4b811b3941626cac47e), but since indigo is EOL, that didn't pass the check either...

Both repos are on Github, so we might just leave the old URL in and rely on Github to do the forward, but that's a bit brittle...

Reply:
The CI script doesn't have this exception. But we can manually override it with just the URL change if that's the only error and it's just the url update. I think a4e78b6 + e84dcee12cf360d95d9d1e795c913f1354078c6f should be good.

Reply:
Alright, I've reverted eceec4d94c64210339e206e346c32232a4bfc156